validation_layers/build.gradle

```diff
@@ -0,0 +1,151 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+def khronosOpenxrSourceDir = file("../thirdparty/openxr-source")
+def khronosOpenxrValidationLayersAndroidLibraryTargetDir = file("${buildDir}/temp/openxr_validation_layers")
+def khronosOpenxrValidationLayersAndroidAssetsTargetDir = file("${buildDir}/temp/openxr_validation_layers_assets")
+def khronosOpenxrValidationLayersAndroidBuildDir = file("${buildDir}/temp/openxr_validation_layers_build")
+
+android {
+    compileSdk versions.compileSdk
+    ndkVersion versions.ndkVersion
+
+    defaultConfig {
+        minSdk versions.minSdk
+        targetSdk versions.targetSdk
+        versionName getReleaseVersion()
```

Comment:
Do the validation layers have their own version?

Reply:
So should this be set to the OpenXR version instead of setting it to the vendors plugin version?